monkey/monkey_island/cc/services/config.py

```diff
@@ -281,3 +309,22 @@ def decrypt_ssh_key_pair(pair, encrypt=False):
             pair['public_key'] = encryptor.dec(pair['public_key'])
             pair['private_key'] = encryptor.dec(pair['private_key'])
         return pair
+
+    @staticmethod
+    def remove_PBA_files():
```

Comment:
move `remove_PBA_files`, `remove_file`, `keep_PBA_files` and related consts to a new python file. config.py is big enough